src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,80 @@
-import React from 'react';
+/* eslint-disable max-len */
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[] | null
+  setMovies: (movies: Movie[]) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [query, setQuery] = useState('');
+  const [selectedMovie, selectMovie] = useState<Movie | null>(null);
+  const [notFound, setNotFound] = useState(false);
```

Comment:
follow convention for the boolean variables names

Reply:
Thank you for informing me about this convention) I was not previously aware of it.